js/country.js

```diff
@@ -1,8 +1,7 @@
 let country = localStorage.getItem("country");
-console.log(country)
+console.log(country);
```

Comment:
remove any unused console.log that you used for testing

Reply:
👍